packages/cli/src/workflows/workflows.services.ts

```diff
@@ -175,12 +183,26 @@ export class WorkflowsService {
 		}
 
 		filter.id = In(sharedWorkflowIds);
-		return Db.collections.Workflow.find({
+
+		if (typeof filter.name === 'string' && filter.name !== '') {
+			filter.name = Like(`%${filter.name}%`);
+		}
+
+		const findManyOptions: FindManyOptions<WorkflowEntity> = {
 			select,
 			relations,
 			where: filter,
-			order: { updatedAt: 'DESC' },
-		});
+			order: { updatedAt: 'ASC' },
+		};
+
+		if (options?.take) {
+			const { skip, take } = WorkflowRepository.toPaginationOptions(options.take, options.skip);
+
+			findManyOptions.skip = skip;
+			findManyOptions.take = take;
+		}
+
+		return Db.collections.Workflow.findAndCount(findManyOptions);
```

Comment:
should we move to repository here?

Reply:
Do you mean this? Because it doesn't seem to work:

```ts
return Container.get(WorkflowRepository).findAndCount(findManyOptions);
```

```
  ● WorkflowService › getMany() › should query for workflows › select

    ServiceNotFoundError: Service with "MaybeConstructable<DataSource>" identifier was not found in the container. Register it before usage via explicitly calling the "Container.set" function or using the "@Service()" decorator.

      203 | 		}
      204 |
    > 205 | 		return Container.get(WorkflowRepository).findAndCount(findManyOptions);
          | 		                 ^
      206 | 	}
      207 |
      208 | 	static async update(
```

Reply:
that's weird. `Container.get(WorkflowRepository)` and `Db.collections.Workflow` should refer to the same object at all times.